pkg/alignchecker/alignchecker_test.go

```diff
@@ -7,12 +7,14 @@ import (
 	"os"
 	"path/filepath"
 	"testing"
+
+	tus "github.com/cilium/tetragon/pkg/testutils/sensors"
```

Comment:
Just a nit but I think I would prefer testutils as an import name here over tus.

Reply:
You mean:
> testutils "github.com/cilium/tetragon/pkg/testutils/sensors"

Isn't that somewhat confusing since there is also a `testutils` package one level up?
There might even be files that need to import both "testutils"  and "testutils/sensors"

Reply:
@kkourt or maybe just leave it as sensors? or maybe sensorutils?

Reply:
How about tusensors? :)

Reply:
Actually, I think I'm going to keep it as is in this PR for consistency, since it already exists in the tree: https://github.com/cilium/tetragon/blob/7cadc51fbafb022fee35d972d4b7a83faf9e9440/pkg/sensors/exec/exec_test.go#L23. We can rename it later.

Reply:
sounds good